### DISCUSSION TITLE
Disable thumbnail preloading on the first opening (with different aspect ratio).

### DISCUSSION THREAD

**Discussion by H852ing** [dimsemenov/PhotoSwipe#2044] [Q&A] [answered]
Hi,
I need your help. I have thumbnails in my project with dimensions of 290x218 pixels (individually edited pictures). When I click on a thumbnail, the full-sized picture appears with dimensions of 667x1000 pixels (vertical picture) or 1500x1000 pixels (horizontal picture).

It works almost perfectly, but there is a delay of about one second after I click on the thumbnail where the thumbnail picture is displayed in the large mode. After this one second, the actual full-sized picture appears and the thumbnail disappears.

Since the thumbnail and the final large picture have different aspect ratios, it looks like the picture is jumping.

My wish is to disable this one-second display of the thumbnail in the full-size frame.

Thank you for your help.

**Comment by dimsemenov** [comment, marked as answer]
You can disable placeholder image via `placeholderSrc` filter or by adjusting `msrc` in your data https://photoswipe.com/data-sources/#custom-html-markup
```
lightbox.addFilter('placeholderSrc', (placeholderSrc, content) => {
  return false;
});
```

**Reply by H852ing** [reply]
Thank you very much.
That's exactly what I'm looking for.
